models/Meta.d.ts

```diff
@@ -0,0 +1,13 @@
+// extend route meta data types
+
+import 'vue-router'
+
+// To ensure it is treated as a module, add at least one `export` statement
```

Comment:
Weird. Even when explicitly declaring a module underneath?

Reply:
Apparently. Copied from the docs.